l2geth/miner/worker.go

```diff
@@ -506,7 +506,7 @@ func (w *worker) mainLoop() {
 				}
 				w.pendingMu.Unlock()
 			} else {
-				log.Debug("Problem committing transaction: %w", err)
+				log.Debug("Problem committing transaction", "msg", err)
```

Comment:
Why this diff?

Reply:
This was a typo in the logline, it is fixed here